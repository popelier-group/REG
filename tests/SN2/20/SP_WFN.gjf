# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.15788000   -0.00006400    0.00017900
 H                 -0.93687200   -0.89462200   -0.55720500
 H                 -0.93682100    0.92993100   -0.49579700
 H                 -0.93695300   -0.03552300    1.05360100
 Br                 1.63931700    0.00001000   -0.00001800
 Cl                -3.22129400    0.00000600   -0.00003800

frame.wfn
