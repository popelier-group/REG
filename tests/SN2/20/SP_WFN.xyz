
6

C       -0.056875    0.233653    1.012605
H       -0.379807   -0.793674    1.004322
H       -0.741187    0.945381    0.582594
H        0.986781    0.400393    0.805861
Br        0.096264   -0.394343   -1.708880
Cl       -0.170223    0.696940    3.020140
