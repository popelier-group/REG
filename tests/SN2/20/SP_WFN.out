 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.20/Gau-20694.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.20/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     20695.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.15788  -0.00006   0.00018 
 H                    -0.93687  -0.89462  -0.55721 
 H                    -0.93682   0.92993  -0.4958 
 H                    -0.93695  -0.03552   1.0536 
 Br                    1.63932   0.00001  -0.00002 
 Cl                   -3.22129   0.00001  -0.00004 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.157880   -0.000064    0.000179
      2          1           0       -0.936872   -0.894622   -0.557205
      3          1           0       -0.936821    0.929931   -0.495797
      4          1           0       -0.936953   -0.035523    1.053601
      5         35           0        1.639317    0.000010   -0.000018
      6         17           0       -3.221294    0.000006   -0.000038
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.076919   0.000000
     3  H    1.076917   1.825586   0.000000
     4  H    1.076923   1.825581   1.825578   0.000000
     5  Br   2.797197   2.783446   2.783350   2.783620   0.000000
     6  Cl   2.063414   2.515826   2.515821   2.515876   4.860611
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.056875    0.233653    1.012605
      2          1           0       -0.379807   -0.793674    1.004322
      3          1           0       -0.741187    0.945381    0.582594
      4          1           0        0.986781    0.400393    0.805861
      5         35           0        0.096264   -0.394343   -1.708880
      6         17           0       -0.170223    0.696940    3.020140
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         150.4628749           0.8488543           0.8488543
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       171.0683111717 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  3.89D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39495438     A.U. after   12 cycles
            NFock= 12  Conv=0.26D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.57338-101.37731 -62.22654 -56.04460 -56.04411
 Alpha  occ. eigenvalues --  -56.04411 -10.08696  -9.29047  -8.43541  -7.05337
 Alpha  occ. eigenvalues --   -7.04629  -7.04629  -6.25939  -6.25766  -6.25766
 Alpha  occ. eigenvalues --   -2.37160  -2.37111  -2.37111  -2.36974  -2.36974
 Alpha  occ. eigenvalues --   -0.65761  -0.54865  -0.50285  -0.30403  -0.30403
 Alpha  occ. eigenvalues --   -0.22448  -0.15453  -0.15453  -0.07016  -0.06896
 Alpha  occ. eigenvalues --   -0.06896
 Alpha virt. eigenvalues --    0.10933   0.11502   0.15187   0.15187   0.16957
 Alpha virt. eigenvalues --    0.17215   0.19328   0.19328   0.20990   0.20990
 Alpha virt. eigenvalues --    0.21187   0.22662   0.22662   0.24780   0.29060
 Alpha virt. eigenvalues --    0.31436   0.31437   0.37243   0.39823   0.54218
 Alpha virt. eigenvalues --    0.54218   0.56075   0.57388   0.59327   0.59327
 Alpha virt. eigenvalues --    0.61962   0.61962   0.68712   0.77128   0.77129
 Alpha virt. eigenvalues --    0.80551   0.81758   0.86285   0.86286   0.87182
 Alpha virt. eigenvalues --    0.87183   0.94855   1.02131   1.02131   1.04256
 Alpha virt. eigenvalues --    1.04256   1.06703   1.16183   1.24264   1.24265
 Alpha virt. eigenvalues --    1.25488   1.54584   1.65320   1.65321   1.79587
 Alpha virt. eigenvalues --    1.90129   1.90129   2.00744   2.18804   2.18805
 Alpha virt. eigenvalues --    2.25455   2.25456   2.32187   2.44634   2.44635
 Alpha virt. eigenvalues --    2.53678   2.63187   2.63187   2.64695   2.72946
 Alpha virt. eigenvalues --    2.72947   3.11297   3.19394   3.85816   3.85817
 Alpha virt. eigenvalues --    4.38006   4.38009   4.38890   4.38891   4.55766
 Alpha virt. eigenvalues --    6.76932   7.78803   7.78808   7.86096   9.84036
 Alpha virt. eigenvalues --   24.03934  25.99649  25.99649  26.40189  48.07928
 Alpha virt. eigenvalues --  215.82316 289.99602 289.99602 290.066411020.84251
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.395521   0.417846   0.417849   0.417841  -0.034770  -0.119689
     2  H    0.417846   0.514593  -0.036377  -0.036386  -0.012230  -0.036441
     3  H    0.417849  -0.036377   0.514585  -0.036385  -0.012236  -0.036438
     4  H    0.417841  -0.036386  -0.036385   0.514603  -0.012213  -0.036445
     5  Br  -0.034770  -0.012230  -0.012236  -0.012213  35.862305   0.043177
     6  Cl  -0.119689  -0.036441  -0.036438  -0.036445   0.043177  17.424187
 Mulliken charges:
               1
     1  C   -0.494599
     2  H    0.188996
     3  H    0.189002
     4  H    0.188985
     5  Br  -0.834033
     6  Cl  -0.238351
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.072384
     5  Br  -0.834033
     6  Cl  -0.238351
 Electronic spatial extent (au):  <R**2>=           1137.4482
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.0894    Y=              0.3679    Z=              1.5944  Tot=              1.6388
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -44.2899   YY=            -45.9486   ZZ=            -77.3085
   XY=              0.4308   XZ=              1.8668   YZ=             -7.6438
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             11.5591   YY=              9.9004   ZZ=            -21.4595
   XY=              0.4308   XZ=              1.8668   YZ=             -7.6438
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              2.6293  YYY=             -8.4208  ZZZ=            -72.5113  XYY=             -0.0468
  XXY=             -1.9302  XXZ=            -10.8608  XZZ=              2.8329  YZZ=            -11.8250
  YYZ=            -12.7263  XYZ=              0.7362
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -78.1171 YYYY=           -141.8075 ZZZZ=          -1501.2426 XXXY=              8.2509
 XXXZ=             36.0450 YYYX=              7.9487 YYYZ=           -147.5542 ZZZX=             44.8714
 ZZZY=           -184.2373 XXYY=            -36.2857 XXZZ=           -235.6203 YYZZ=           -254.3785
 XXYZ=            -48.1462 YYXZ=             11.8531 ZZXY=              5.2085
 N-N= 1.710683111717D+02 E-N=-7.695416088945D+03  KE= 3.069986649294D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE531\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.15788,-0.000064,0.000179\H,0,-0.936872,-0.894622,-0.557205\H,0,
 -0.936821,0.929931,-0.495797\H,0,-0.936953,-0.035523,1.053601\Br,0,1.6
 39317,0.00001,-0.000018\Cl,0,-3.221294,0.000006,-0.000038\\Version=ES6
 4L-G16RevC.01\State=1-A\HF=-3074.3949544\RMSD=2.569e-09\Dipole=-0.6447
 359,-0.000055,0.0001541\Quadrupole=-17.344282,8.6721747,8.6721073,0.00
 00541,-0.000252,0.0000057\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 THE SOLUTION TO A PROBLEM CHANGES THE PROBLEM.

                                 -- JOHN PEERS
           PAUL DICKSON'S "THE OFFICIAL RULES"
 Job cpu time:       0 days  0 hours  0 minutes 27.5 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:50:16 2020.
